Typescript import on v5.0.0

## What did you do? 
I was trying to use the last release (v.5.0.0) to add a loader.
I am using `"react": "^16.8.0"` and `"typescript": "^3.7.4"`.
In my component, I did 

```typescript
import React from "react";
import ContentLoader from "react-content-loader";

interface IProps {}

const Loader: React.FC<IProps> = () => {
  return (
    <ContentLoader>
      <rect x="0" y="0" rx="5" ry="5" width="70" height="70" />
      <rect x="80" y="17" rx="4" ry="4" width="300" height="13" />
      <rect x="80" y="40" rx="3" ry="3" width="250" height="10" />
    </ContentLoader>
  );
};

export default Loader;
```
as seen [in this CodeSandbox](https://codesandbox.io/s/32733wppk5), 
but changing the version to `5.0.0` instead of `4.2.0`.

## What did you expect to happen?
I expected my build to work as demonstrated.

## What happened actually?
Actually, i get a TypeScript error when building : 

```
src/components/ImgLoader.tsx:2:27 - error TS7016:   
Could not find a declaration file for module 'react-content-loader'.   
'/home/node_modules/react-content-loader/dist/react-content-loader.cjs.js'   
implicitly has an 'any' type.
Try `npm install @types/react-content-loader` if it exists or add a new   
declaration (.d.ts) file containing `declare module 'react-content-loader';`

import ContentLoader from 'react-content-loader'
```

It seems related to the fact there is no more `index.ts` file in `src` folder.

### Which versions of react-content-loader, and which browser are affected by this issue?
| package                   | version |
|----------------------------|-----------|
| react                         | 16.8.0  |
| typescript                  | 3.7.4    |
| react-content-loader | 5.0.0    |

--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
   "main": "dist/react-content-loader.cjs.js",
   "module": "dist/react-content-loader.es.js",
   "jsnext:main": "dist/react-content-loader.es.js",
-  "types": "dist/index.d.ts",
+  "types": "dist/web/index.d.ts",
   "bugs": {
     "url": "https://github.com/danilowoz/react-content-loader/issues"
   },

